### ISSUE TITLE
Mines are producing ore without getting food

### ISSUE THREAD

**Issue by stumpdk** [jsettlers/settlers-remake#668]
Experienced in the nightly build (8ab28fb)

**Comment by stumpdk** [comment]
When a mine is built, it produces 8 ore resources without food. Then it produces resources as normal.

**Comment by stumpdk** [comment]
A mine produces 10 ore resources for one food resource.
Hmm, sounds like it is in purpose...

**Comment by stumpdk** [comment]
[The feedWorkPackages is set to 10](https://github.com/jsettlers/settlers-remake/blob/master/jsettlers.logic/src/main/java/jsettlers/logic/buildings/workers/MineBuilding.java#L40)

Is there a particular reason for that?

**Comment by andreas-eberle** [comment]
@codingberlin: Do you know the exact amounts of ore each type of mine should produce for the specific type of food?

**Comment by andreas-eberle** [comment]
@stumpdk: The initial value is set to 10 to have the mine produce 10 ores without needing food. This is important for game rushs and is the same configuration like in the original game.

**Comment by codingberlin** [comment]
@andreas-eberle yes, in the comments of issue https://github.com/jsettlers/settlers-remake/issues/151 are the results of the Test. Each mine has its own corresponding type of food.

By the way: the 10 ore at start are for low goods games in the original game where you start without food and very few tools.

**Comment by stumpdk** [comment]
I can confirm @codingberlin's observations, that a mine makes 10 checks for mine resources per food resource, and that the first check is performed even without the mine having food.

I can add that one check takes 13 game seconds.
